services/app-api/src/resolvers/contract/contractRevisionResolver.ts

```diff
@@ -0,0 +1,27 @@
+import { packageName } from 'app-web/src/common-code/healthPlanFormDataType'
```

Comment:
On the frontend I get an auth error (signing in doesn't go through) Check Auth Failed. ApolloError: Response not successful: Received status code 502. Until this path is updated to a relative on2 `../../../../app-web/src/common-code/healthPlanFormDataType`

Reply:
pushed a fix for that, I can log in locally lmk if you still can't